### DISCUSSION TITLE
[Feature] support playframework v3

### DISCUSSION THREAD

**Discussion by pjfanning** [apache/skywalking#11881] [General]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Play Framewwork v3 uses Apache Pekko instead of Akka because Akka is no longer OSS.

https://www.playframework.com/documentation/3.0.x/General#How-Play-Deals-with-Akkas-License-Change 

### Are you willing to submit a pull request to implement this on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by pjfanning** [comment]
@wu-sheng I think this is blocked by https://github.com/play2-maven-plugin/play2-maven-plugin/issues/207

I don't know much about SkyWalking and its Play Scenario but it appears to rely on this community plugin (which isn't part of the Play Framework itself).

**Comment by wu-sheng** [comment]
I don't know akka much, neither Pekko for code details.

Why a maven plugin is a block of plugin build? This seems strange.

And FYI, issue is for tracking dev process, if this is not certain, it should be a discussion.(I am going to move this to there).

**Reply by pjfanning** [reply]
https://github.com/apache/skywalking-java/pull/671 implements the play3 plugin. What is incomplete is the test scenario.

**Reply by wu-sheng** [reply]
OK. When you have a pull request to review(including discussions), please send the pull request directly. You don't need to maintain one thing in two places.

**Reply by wu-sheng** [reply]
Issue and discussion majorly aim to things not ready to review through a pull request.
